src/realm/index_string.hpp

```diff
@@ -226,7 +226,7 @@ class StringIndex {
     struct NodeChange {
         size_t ref1;
         size_t ref2;
-        enum ChangeType { none, insert_before, insert_after, split } type;
+        enum class ChangeType { change_None, change_InsertBefore, change_InsertAfter, change_Split } type;
```

Comment:
I think it is overdoing things to both have the enum class `ChangeType` and have `change_`  in the enum values.

Reply:
Fair enough, I removed the enum class.